index.html

```diff
@@ -4,19 +4,33 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>301 Lab 1</title>
+
+  <link rel="stylesheet" type="text/css" href="./css/reset.css">
+  <link rel="stylesheet" href="./css/base.css" type="text/css">
+  <link rel="stylesheet" href="./css/layout.css" type="text/css">
+  <link rel="stylesheet" href="./css/module.css" type="text/css">
+
 </head>
 
 <body>
 
-  <div></div>
-  <div></div>
-  <div></div>
-  <div></div>
-  <div></div>
-  <div></div>
-  <div></div>
-  <div></div>
-  <div></div>
+  <section id="mainContainer">
+    <div id="a"></div>
```

Comment:
Since the 'a' box seems likely to be the header for the web page you could use the <header> tag instead of depending on these nested div tags and their id property.